### DISCUSSION TITLE
getBlock return an Error

### DISCUSSION THREAD

**Discussion by amncjf** [filecoin-project/lotus#10903] [General Q&A]
I in https://api.hyperspace.node.glif.io/rpc/v1 call getBlock (294023) interface, return an Error: Error: requested epoch was a null round, Recommended return:
{
  ...prevBlock,
  parentHash: prevBlock.hash,
  blockNumber: prevBlock.blockNumber+1,
}

**Comment by TippyFlitsUK** [comment]
Please use the issue templates provided to submit new tickets @amncjf!
I would also suggest asking this in https://github.com/filecoin-project/ref-fvm
Many thanks 🙏

**Comment by amncjf** [comment]
I use new issue templates to resubmit the issue, also gave the issue to https://github.com/filecoin-project/ref-fvm
Thanks🤝
